duckduckgo/iOS
# Remove App Icon experiment

Task/Issue URL: https://app.asana.com/0/856498667320406/1165279504863374

**Description**:
Remove App Icon onboarding experiment.

**Steps to test this PR**:
1. Run the app.
2. Make sure onboarding does not include picking App Icon step. 
3. Icon switching through settings should work.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
